core/src/main/java/com/linecorp/armeria/common/metric/BuiltInMetricLabel.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.metric;
+
+import com.linecorp.armeria.server.PathMapping;
+
+/**
+ * {@link MetricLabel} used by {@link MetricKeyFunction#ofDefault(String...)}.
+ */
+public enum BuiltInMetricLabel implements MetricLabel {
```

Comment:
I prefer `BuiltInMetricLabel` because `Default` gives an impression that it's the *default* implementation of `MetricLabel` a user is encouraged to use. It is also sort of consistent with `BuiltInProperty` enum in `armeria-logback`.